src/routes/console/project-[project]/functions/templates/+page.svelte

```diff
@@ -82,23 +79,10 @@
     </div>
     <div class="grid-300px-1fr u-margin-block-start-24">
         <section>
-            <div
-                class="input-text-wrapper is-with-end-button u-width-full-line u-max-width-500 u-margin-block-start-12"
-                style="--amount-of-buttons:1">
-                <input
-                    bind:this={searchElement}
-                    on:input={applySearch}
-                    type="search"
-                    placeholder="Search templates" />
-                <div class="icon-search" aria-hidden="true" />
-                <button
-                    on:click={clearSearch}
-                    class="button is-text is-only-icon"
-                    aria-label="Clear search"
-                    style="--button-size:1.5rem;">
-                    <span class="icon-x" aria-hidden="true" />
-                </button>
-            </div>
+            <InputSearch
+                placeholder="Search templates"
+                on:clear={clearSearch}
+                on:change={applySearch} />
```

Comment:
why dont we need the debouncing anymore?

Reply:
It's already the InputSearch component